src/frontend/screens/WebView/index.tsx

```diff
@@ -217,8 +217,24 @@ export default function WebView({ store }: Props) {
         }
       }
 
-      webview.addEventListener('dom-ready', loadstop)
+      const onerror = (error: Electron.DidFailLoadEvent) => {
+        const { errorCode, validatedURL } = error
+        // Connection refused
+        if (
+          errorCode === -102 &&
```

Comment:
maybe we can ignore this check? in case it fails to load adtraction for any reason, not just connection refused

Reply:
huh, maybe you are right. As long as the url matches we should be good to go